vaccine_feed_ingest/runners/ny/am_i_eligible_covid19vaccine_gov/normalize.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import re
+import sys
+
+CITY_RE = re.compile(r"^([\w ]+), NY$")
+# the providerName field smells like it's being parsed from someplace else,
+# a good number of them have leading \u1d42 and/or *, which we want to clean.
+# there's a bunch with a city name in them, but no real pattern to it, so
+# we'll leave that for now.
+NAME_CLEAN_RE = re.compile("^[\u1d42*]+")
+
+
+def normalize(site_blob: dict, timestamp: str) -> dict:
+    """
+    sample entry:
+
+    {"providerId": 1013, "providerName": "\u1d42**York College - Health and Physical Education Complex - Queens", "vaccineBrand": "Pfizer", "address": "Jamaica, NY", "availableAppointments": "Y", "isShowable": true, "lastUpdated": "2021-04-23T20:04:24"} # noqa: E501
+    """
+    name = NAME_CLEAN_RE.sub("", site_blob["providerName"]).strip()
```

Comment:
It looks to me like the normalized output has a few of the leading `\u1d42`s left in, e.g., `"providerName": "\u1d42Suffolk CCC - Brentwood",`.

Should we consider a regex more like `^(\\u1d42)?\**`?

Reply:
I'm actually not seeing that on my end.

```
ono-sendai:runners bonnerd$ rm -rf out
ono-sendai:runners bonnerd$ poetry run vaccine-feed-ingest fetch ny/am_i_eligible_covid19vaccine_gov
ono-sendai:runners bonnerd$ poetry run vaccine-feed-ingest parse ny/am_i_eligible_covid19vaccine_gov
ono-sendai:runners bonnerd$ poetry run vaccine-feed-ingest normalize ny/am_i_eligible_covid19vaccine_gov
ono-sendai:runners bonnerd$ cat out/ny/am_i_eligible_covid19vaccine_gov/normalized/2021-04-24T11\:07\:02/data.normalized.ndjson | jq '.name' | rg -i brentwood
"Suffolk CCC - Brentwood"
```